src/elements/content-sidebar/Sidebar.js

```diff
@@ -112,11 +119,13 @@ class Sidebar extends React.Component<Props, State> {
         const { file_version: currentVersion } = file;
         const fileVersionSlug = currentVersion ? `/${currentVersion.id}` : '';
 
+        const urlPrefix = this.getUrlPrefix(history.location.pathname);
```

Comment:
Minor, but this is probably simple enough to inline: `const basePath = pathname.substring(1).split('/')[0];`